src/pykeen/datasets/ea/wk3l.py

```diff
@@ -103,7 +103,7 @@ def _load_df(self, key: Union[None, EASide, Tuple[EASide, EASide]], **kwargs) ->
             encoding="utf8",
             dtype=str,
             **kwargs,
-        )
+        ).dropna()
```

Comment:
is this because there are missing records, or because the text `nan` is used in some places? Maybe we need the same fix as in #883

Reply:
I updated the issue description with the relevant information

Reply:
I used the same way as https://github.com/pykeen/pykeen/pull/883 in [4f6a97b](https://github.com/pykeen/pykeen/pull/907/commits/4f6a97b419ef074e84e8741efeed5bb28ba4e24e)